infrastructure/terraform/per_workspace/modules/etl/sds/main.tf

```diff
@@ -432,3 +440,36 @@ module "bulk_trigger_notification" {
   filter_prefix = "${local.bulk_trigger_prefix}/"
   filter_suffix = ".ldif"
 }
+
+module "proxy_lambda_executor" {
+  source = "./proxy-executor/"
+
+  executor_name         = "input"
```

Comment:
maybe keep the same naming convention all the way down, so that the lambda has the same name as the module. Would help to draw the developers eye to parts of the codebase that are related

Reply:
imo not immediately clear that this needs to be a parameter. I think unless the parameters can't be hard-coded into the module probably best to hard code them into the module as by definitions this isn't a re-use module (i.e. we only want one state locking mechanism for this etl). Might be an idea to have variables like this as `locals` instead inside the module, especially if they're repeatedly used in the module